tailnet/conn.go

```diff
@@ -247,6 +248,17 @@ func NewConn(options *Options) (conn *Conn, err error) {
 		server.lastMutex.Unlock()
 		server.sendNode()
 	})
+	magicConn.SetDERPForcedWebsocketCallback(func(region int, reason string) {
+		server.logger.Debug(context.Background(), "derp forced websocket", slog.F("region", region), slog.F("reason", reason))
+		server.lastMutex.Lock()
+		if server.lastDERPForcedWebsockets[region] == reason {
+			server.lastMutex.Unlock()
+			return
```

Comment:
If the reason didn't change, I figured there was no reason to update the node again, because the data wouldn't change.

Reply:
You're probably right, I was just trying to play devils advocate and be overly cautious in case it could lead to a scenario where we should sendNode and aren't. 😄

Reply:
Ahh, I see!